tfx_addons/example_filter/executor.py

```diff
@@ -0,0 +1,367 @@
+                            examples_list.append(data)
+
+                custom_extractors = udf_utils.try_get_fn(
+                    exec_properties=exec_properties, fn_name='custom_extractors')
+                extractors = None
+                if custom_extractors:
+                    extractors = custom_extractors(
+                        eval_shared_model=eval_shared_model,
+                        eval_config=eval_config,
+                        tensor_adapter_config=tensor_adapter_config)
+
+                try:
+                    examples_list = filter(input_dict[standard_component_specs.FILTER_FUNCTIONS_KEY][0], examples_list)
+
+                    (examples_list | 'FlattenExamples' >> beam.Flatten()
+                     | 'ExtractEvaluateAndWriteResults' >> (tfma.ExtractEvaluateAndWriteResults(
+                                eval_shared_model=models[0] if len(models) == 1 else models,
+                                eval_config=eval_config,
+                                extractors=extractors,
+                                output_path=output_uri,
+                                slice_spec=slice_spec,
+                                tensor_adapter_config=tensor_adapter_config)))
+                except Exception as e:
+                    print('exc', e)
+
+        logging.info('Evaluation complete. Results written to %s.', output_uri)
+
+        if not run_validation:
+            # TODO(jinhuang): delete the BLESSING_KEY from output_dict when supported.
+            logging.info('No threshold configured, will not validate model.')
+            return
+        # Set up blessing artifact
+        blessing = artifact_utils.get_single_instance(
+            output_dict[standard_component_specs.BLESSING_KEY])
+        blessing.set_string_custom_property(
+            constants.ARTIFACT_PROPERTY_CURRENT_MODEL_URI_KEY,
+            artifact_utils.get_single_uri(
+                input_dict[standard_component_specs.MODEL_KEY]))
+        blessing.set_int_custom_property(
+            constants.ARTIFACT_PROPERTY_CURRENT_MODEL_ID_KEY,
+            input_dict[standard_component_specs.MODEL_KEY][0].id)
+        if input_dict.get(standard_component_specs.BASELINE_MODEL_KEY):
+            baseline_model = input_dict[
+                standard_component_specs.BASELINE_MODEL_KEY][0]
+            blessing.set_string_custom_property(
+                constants.ARTIFACT_PROPERTY_BASELINE_MODEL_URI_KEY,
+                baseline_model.uri)
+            blessing.set_int_custom_property(
+                constants.ARTIFACT_PROPERTY_BASELINE_MODEL_ID_KEY, baseline_model.id)
+        if 'current_component_id' in exec_properties:
+            blessing.set_string_custom_property(
+                'component_id', exec_properties['current_component_id'])
+        # Check validation result and write BLESSED file accordingly.
+        logging.info('Checking validation results.')
```

Comment:
Why is exampleFilter doing model validation? Let's remove anything that is not related to the example filter.